scripts/contentScript.js

```diff
@@ -426,8 +433,8 @@ const orchestrator = (tab_url, websiteUrl, userId) => {
 			// for showing tracked repo of a organization 
 			if (urlObj[4] === 'workspace' && urlObj[5] === 'repositories') {
 				const workspace_slug = urlObj[3];
-				const body = { "org": workspace_slug, "userId": userId, "is_github": false }
-				const url = `${backendUrl}/setup/repos`;
+				const body = { "workspace_slug": workspace_slug, "userId": userId, "is_github": false }
+				const url = `${backendUrl}/bitbucket/setup/repos`;
 
 				const response = await apiCall(url, body);
 				updateTrackedReposInBitbucketOrg(response.repos, websiteUrl);
```

Comment:
Reviewed it , apart from changing api endpoints, you changed the function getTracked repos and now it'll will work for both gitHub and bitBucket if api is right but it's not called for bitBucket.

Reply:
Got it